### PR TITLE
PUB-1050 Admin Login Page

### PR DESCRIPTION
**Pull request by junaidiqbalmoj**
### JIRA link ###

https://tools.hmcts.net/jira/browse/PUB-1050

### Change description ###

Create Admin login screen. Made changes so that only admin can access the admin screen.

Image 06-05-2022 at 14 52

**Does this PR introduce a breaking change?** (check one with "x")
[ ] Yes
[x] No

### CODE REVIEW

**Review by junaidiqbalmoj — COMMENTED** [review]
Fix the existing tests and add unit tests.

**Review by furnivall — DISMISSED** [review]
LGTM

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

**Review by ChrisS1512 — APPROVED** [review]

File: src/main/authentication/adminAuthentication.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const adminRolesList = ['INTERNAL_SUPER_ADMIN_CTSC', 'INTERNAL_SUPER_ADMIN_LOCAL', 'INTERNAL_ADMIN_CTSC', 'INTERNAL_ADMIN_LOCAL'];
+
+export class AdminAuthentication {
+  public isAdminUser(req: any): boolean {
+    if(req.user) {
+      const userInfo = req.user['_json'];
```
[the remark this answers is not in the document]
Comment (junaidiqbalmoj) [reply src/main/authentication/adminAuthentication.ts:6]: Double check again, user object does not contain extension_UserRole attribute in it. Custom user attributes like extension_UserRole, family_name etc are coming in _json.

**Review by ChrisS1512 — APPROVED** [review]

**Review by furnivall — APPROVED** [review]
LGTM :)

**Review by kurt-uk — APPROVED** [review]

**Pull request merged**